index.bs

```diff
@@ -478,7 +478,9 @@ When this method is invoked, the user agent MUST execute the following algorithm
 1. Let |options| be the value of |options|["{{CredentialRequestOptions/scoped}}"].
 
 1. If any of the {{ScopedCredentialEntity/name}} member of |options|.{{MakeCredentialOptions/rp}}, the
-    {{ScopedCredentialEntity/name}} member of |options|.{{MakeCredentialOptions/user}}, or the {{ScopedCredentialEntity/id}}
+    {{ScopedCredentialEntity/name}} member of |options|.{{MakeCredentialOptions/user}},
+    the {{ScopedCredentialUserEntity/displayName}} member of |options|.{{MakeCredentialOptions/user}},
+    or the {{ScopedCredentialEntity/id}}
```

Comment:
The problem with not requiring this information is that the RP doesn't know whether the authenticator needs it or not.  I think the PR does what we need, as-is.